index.html

```diff
@@ -0,0 +1,171 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <link rel="stylesheet" href="./css/style.css" />
+    <title>Youtube clone</title>
+  </head>
+  <body>
+    <header class="nav-bar">
+      <img src="images/download.png" class="logo" />
+      <form class="search-field">
+        <input type="text" class="search-input" placeholder="Search" />
+        <button type="submit" class="search-btn">
+          <i class="fa fa-search grey-icon"></i>
+        </button>
+      </form>
+      <i class="fa fa-upload upload-icon grey-icon" aria-hidden="true"></i>
```

Comment:
Navigation bar should have at least three icons on the rightmost part of the navbar.

Reply:
Thanks for the feedback, it has been resolved. kindly confirm if it satisfies your request